website/docs/project_settings/settings_project_kitsu.md

```diff
@@ -0,0 +1,13 @@
+---
+id: settings_project_kitsu
+title: Project KItsu Setting
+sidebar_label: Kitsu
+---
+
+# Entities naming pattern
+Allows to organize hierarchically the Episodes, Sequences et Shots into their folders.
```

Comment:
```suggestion
Allows to organize hierarchically the Episodes, Sequences and Shots into their folders.
```

By just reading this docs with no more information I'm not entirely sure how this "allows me to hierarchically organize into folders. The settings (aside form being Episodes, Sequences and Shots) seem to have no hierarchical setting tweaks at all? It only seems to set some naming. This might be worth elaborating on what this does when changing the values in the settings. What changes if I change Episode to `Hello_World`. And also what does the `##` do in the name, is the `##` required?, etc.